modules/ROOT/pages/creating-a-skin.adoc

```diff
@@ -67,6 +67,30 @@ _Do not edit these files directly_. Instead, use them as references when creatin
 * `+modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less+`
 
 The typical workflow involves copying variables from the theme folder into the skins `skin.less` file and modifying them as needed.
+
+NOTE: Split buttons use `.tox-split-button__main` and `.tox-split-button__chevron` CSS classes. If your skin targets split button components, you may need to update your CSS selectors.
+
+.Example: Updating split button CSS selectors
+[source,css]
+----
+/* Before: targeting the composite split button */
+.tox-split-button {
+  /* Previous styling approach */
+}
+
+/* After: targeting individual components */
+.tox-split-button__main {
+  /* Main button styling */
+  background-color: #blue;
+  border-radius: 4px 0 0 4px;
+}
+
+.tox-split-button__chevron {
+  /* Chevron button styling */
+  background-color: #darkblue;
+  border-radius: 0 4px 4px 0;
+}
+----
```

Comment:
IMO this is only applicable in TinyMCE 8.0 as a breaking change.  It can also slip into previous versions when backporting